people_server/src/SimplePocoHandler.h

```diff
@@ -0,0 +1,39 @@
+// https://github.com/RPG-18/rabbitmq-cpp-tutorials/blob/master/src/SimplePocoHandler.h
+
+#pragma once
+
+#include <memory>
+#include <amqpcpp.h>
+
+
+class SimplePocoHandlerImpl;
+class SimplePocoHandler: public AMQP::ConnectionHandler
+{
+public:
+    static constexpr size_t BUFFER_SIZE = 8 * 1024 * 1024; //8Mb
+    static constexpr size_t TEMP_BUFFER_SIZE = 1024 * 1024; //1Mb
```

Comment:
Did you handle error for situation when buffer size would be too low?

Reply:
The code is from here:
https://github.com/RPG-18/rabbitmq-cpp-tutorials/blob/master/src/SimplePocoHandler.h
https://github.com/RPG-18/rabbitmq-cpp-tutorials/blob/master/src/SimplePocoHandler.cpp

Buffer is limited by 8Mb, but I think that it is very big number. I think it is out of scope of our app anyway.